tools/ld-chroma-decoder/palcolour.cpp

```diff
@@ -372,6 +374,36 @@ void PalColour::detectBurst(LineInfo &line, const quint16 *inputData)
     line.bq /= burstNorm;
 }
 
+// Perform analog-style noise coring.
+void PalColour::doYNR(double *Yline)
+{
+    // High-pass filter for Y
+    auto yFilter(f_nr);
+
+    const double irescale = (videoParameters.white16bIre - videoParameters.black16bIre) / 100;
+
+    double nr_y = configuration.yNRLevel * irescale;
+
+    double hplinef[videoParameters.fieldWidth];
+
+    for (qint32 h = 0; h <= videoParameters.fieldWidth; h++) {
```

Comment:
< not <=, else it'll write off the end of the array. (The NTSC code has the same bug, looking at it.)